manifests/config.pp.try

```diff
@@ -0,0 +1,150 @@
+# == Class: confluence
+#
+# Install confluence, See README.md for more.
+#
+class confluence::config(
+  $tomcat_port          = $confluence::tomcat_port,
```

Comment:
`config.pp.try`? Is this an accidentally committed file?

Reply:
yes